src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs

```diff
@@ -1,54 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I'll do the `Obsolete` attribute in a follow up. It's not particularly related to this change.